packages/-ember-data/tests/integration/adapter/rest-adapter-test.js

```diff
@@ -2809,22 +2858,24 @@ test('createRecord - sideloaded records are pushed to the store', function(asser
   });
 });
 
-testInDebug(
-  'warns when an empty response is returned, though a valid stringified JSON is expected',
-  function(assert) {
-    server.post('/posts', function() {
-      return [201, { 'Content-Type': 'application/json' }, ''];
-    });
+if (hasjQuery) {
```

Comment:
I've shamefully opted out of this last rest adapter integration test for fetch scenario. I've not been able to make it pass. The network call is not catched by pretender. I suspect an issue in Pretender.

Reply:
Seems fine to me, but would you mind making an issue to dig into it?

Reply:
Alright 👍 
Opened here: https://github.com/emberjs/data/issues/6083